source/configuration_manual/service_configuration.rst

```diff
@@ -187,12 +195,12 @@ auth
 ^^^^^
 The master auth process. There are 4 types of auth client connections:
 
-   * **client**: Only :ref:`sasl` authentication is allowed. This can be safely exposed to entire world.
+   * **auth**: Only :ref:`sasl` authentication is allowed. This can be safely exposed to entire world.
    * **userdb**: userdb lookups and passdb lookups (without the password itself) can be done for any user, and a list of users can be requested. This may or may not be a security issue. Access to userdb lookup is commonly needed by dovecot-lda, doveadm and other tools.
    * **login**: Starts a two phase user login by performing authenticating (same as`client` type). Used by login processes.
    * **master**: Finishes the two phase user login by performing a userdb lookup (similar to "userdb" type). Used by post-login processes (e.g. imap, pop3).
 
-With UNIX listeners the client type is selected based on the filename after the last ``-`` in the filename. For example ``anything-userdb`` is of `userdb` type. The default type is `client` for inet listeners and unrecognized UNIX listeners. You can add as many client and userdb listeners as you want (and you probably shouldn't touch the login/master listeners).
+For Dovecot versions past v3.0, the listener type is configured explicitly using the **type** field. For older versions the client type is selected based on the (file)name after the last ``-`` in the name. For example ``anything-userdb`` is of `userdb` type. The default type is `auth` for inet listeners and unrecognized UNIX listeners. You can add as many client and userdb listeners as you want (and you probably shouldn't touch the login/master listeners).
```

Comment:
`.. versionchanged:: v2.4.0;v3.0.0 The listener type is ..`

Reply:
changed